bin/commands/internal/start/index.ts

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
This file isnt in use yet, I could either remove it or finish it. Currently only `zwe internal start prepare` uses the TS code.
This file is "excluded" in the TS build script as a result.